app/main.py

```diff
@@ -1,6 +1,34 @@
+import requests
+from pprint import pprint
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+URL = "http://api.weatherapi.com/v1/current.json"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    api_key = os.getenv("API_KEY")
+
+    params = {
+        "key": api_key,
+        "q": os.getenv("CITY_NAME")
+    }
+
+    try:
+        response = requests.get(URL, params=params)
+        response.raise_for_status()
+    except requests.exceptions.HTTPError as e:
+        print(e)
+        return
```

Comment:
This is more of a recommendation. It's good that you are handling exceptions that might occur when making a request. However, you should also consider logging the exception error message, not just printing it. This way, you can keep track of the errors that occur, which can be helpful for debugging purposes.